src/natcap/invest/urban_nature_access.py

```diff
@@ -0,0 +1,315 @@
+import os
+import logging
+import shutil
+import tempfile
+
+import pygeoprocessing
+import taskgraph
+from osgeo import gdal
+from osgeo import osr
+import numpy
+
+from . import validation
+from . import spec_utils
+from . import utils
+from .spec_utils import u
+from natcap.invest import MODEL_METADATA
+
+LOGGER = logging.getLogger(__name__)
+UINT32_NODATA = int(numpy.iinfo(numpy.uint32).max)
+FLOAT32_NODATA = float(numpy.finfo(numpy.float32).max)
+ARGS_SPEC = {
+    'model_name': MODEL_METADATA['urban_nature_access'].model_title,
+    'pyname': MODEL_METADATA['urban_nature_access'].pyname,
+    'userguide_html': MODEL_METADATA['urban_nature_access'].userguide,
+    'args_with_spatial_overlap': {
+        'spatial_keys': [],
```

Comment:
Do we know yet which inputs will need to overlap in space?

Reply:
Pretty sure we do!  The first stab at it given what we know about things right now is in a1cea270